Utilities/JIT.cpp

```diff
@@ -109,7 +109,7 @@ extern void jit_finalize()
 
 	for (auto&& t : s_unfire)
 	{
-		mem.deregisterEHFrames(std::get<0>(t), std::get<1>(t), std::get<2>(t));
+		mem.deregisterEHFrames();
```

Comment:
Any reason to call `.deregisterEHFrames()` more than once? If not maybe drop `s_unfire`.

Reply:
It's just to make it compile. deregisterEHFrames doesnt even do anything anymore, because LLVM took it out. There needs to be custom EHFrame management.